src/app/view/perspectives/chart/sorter/creation-date-sorter.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {DocumentModel} from '../../../../core/store/documents/document.model';
+
+export class CreationDateSorter {
+
+  public sortData(documents: DocumentModel[]): DocumentModel[] {
+    if (documents) {
+      return this.copy(documents).sort((a, b) => {
+        // return a.creationDate.localeCompare(b.creationDate);
+        return 1;
+      });
+
+    } else {
+      return [];
+    }
+  }
+
+  private copy(documents: DocumentModel[]): DocumentModel[] {
+    return documents.slice();
```

Comment:
Just wondering. Why do we need this method here? You can just call `documents.slice()` in the method above.